src/components/search.tsx

```diff
@@ -36,13 +36,18 @@ export default class Search extends React.Component<Props> {
     let inputId = `ars_search_${this.id}`
     let listId = `ars_search_list_${this.id}`
 
+    let autoCompleteOpts = {}
+    if (!autoComplete) {
+      autoCompleteOpts['autoComplete'] = 'none'
```

Comment:
what about`'off'` instead of `'none'`, based on https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values?

Reply:
Yup, I think thats right.